Tests/UIKitTests/UITableViewExtensionsTests.swift

```diff
@@ -133,6 +133,16 @@ final class UITableViewExtensionsTests: XCTestCase {
 		XCTAssertNotNil(cell)
 	}
 	#endif
+    
+    #if os(iOS)
```

Comment:
Ditto

Reply:
Will update

Reply:
Reverted due to issues with test cases being run on tvOS. Based pattern on already existing test cased on UITableViewExtensionTests